R/distanceMatrix.R

```diff
@@ -23,22 +25,47 @@ distDTW <- function (q, t = NULL, labels = NULL, step_pattern = dtw::symmetric2)
   for (i in 1:(n - 1)) {
     for (j in (i + 1):n) {
       if (is.null(t))
-        d[n * (i - 1) - i * (i - 1)/2 + j - i] <- DTW(
+        if(normalized){
```

Comment:
You should follow the code style of the package.
```
if (condition) {
} else {
}
```
Pay attention to where spaces and new lines are.